bot.py

```diff
@@ -4,14 +4,15 @@
 #from commands.leaderboard import leaderboard, LEADERBOARD_CHANNEL
 from commands.twitch import twitch, TWITCH_CHANNEL
 
-import json, os, sys
+import json, os, sys, configparser
 
 import discord
 import asyncio
 
 import commands
 
 client = discord.Client()
+config = configparser.ConfigParser()
```

Comment:
Should move all config parsing logic here.

Reply:
All config parser logic is now in a file configstartup.py - I wanted to keep it separate from bot.py